tasks/main.yml

```diff
@@ -72,18 +83,86 @@
     - testing is defined
     - letsencrypt_cert.stat.exists == False
 
+- name: Create self-signed root certificate (CA), if testing.
+  command: >
+    openssl req -x509 -new -nodes -subj '/CN=lili' -days 30
+    -newkey rsa:4096 -sha256 -keyout /etc/letsencrypt/live/CA/rootCA.key
+    -out /etc/letsencrypt/live/CA/rootCA.pem
+  args:
+    creates: /etc/letsencrypt/live/CA/rootCA.pem
+  ignore_errors: yes
+  when:
+    - testing is defined
+    - letsencrypt_cert.stat.exists == False
```

Comment:
With the condition `letsencrypt_cert.stat.exists == False` we're only checking that the cert for `{{ domain }}` exists so if the **CA** was already created this command is going to overwrite it. Even if it doesn't the condition must be related to the file it creates. You could use

```yaml
args:
    creates: /path/to/file
```
in the **command** task so it's only executed if the *rootCA.key* doesn't already exist.